### DISCUSSION TITLE
What is direction in Vote (Like)?

### DISCUSSION THREAD

**Discussion by myConsciousness** [bluesky-social/atproto#648] [Dev Debugging] [answered]
Hi amazing developers,

I saw an Issue where the current VOTE is refactored to LIKE. So the question arises, will the Enum called direction that is currently associated with VOTE be removed?

This direction is an enumeration of "up" and "down" and is reminiscent of a Reddit-like poll. However, in the current UI in Bluesky, this is a simple "like" or "not like" expression.

If VOTE is refactored to LIKE, will the concept of DIRECTION remain? Or does "down" in direction simply mean to remove Like?

Thank you.

**Comment by pfrazee** [comment, marked as answer]
This week we're going to execute on this refactor (which I should emphasize is a one-time thing that is only possible during this pre-release era).

The `app.bsky.feed.vote` type will be replaced with `app.bsky.feed.like` and no direction field will remain. Implicitly, the "up" direction is a like.
